R/bbr.R

```diff
@@ -321,9 +321,22 @@ bbi_help <- function(.cmd_args=NULL) {
 #'   model. `submit_model()` and `submit_models()` also support passing a
 #'   configuration file via the `.config_path` argument.
 #'
+#'  **Notes on `.nonmem_dir`**
+#'
+#'   `bbi` tries to detect subdirectories for different NONMEM versions, along
+#'   with a license file and executable within the subdirectory. `.nonmem_dir`
+#'   should therefore point to the *parent* directory containing the executable.
```

Comment:
Hmm, I'm not sure what "the executable" is here (I know you expand on it a bit in the next paragraph, but I'm explaining how I'd be confused by it as a reader).  The parent directory that should be passed as `.nonmem_dir` will typically just contain (assuming contain means first-level contents) one or more directories for the NONMEM installation.

Even _within_ a NONMEM installation's `run/` directory, I think saying "the executable" would be confusing because there are several executables.  (The one bbi cares about is `nmfe*`, but I don't think we should get into that level of detail here.)

Reply:
I was trying to reword your comment [here](https://github.com/metrumresearchgroup/bbr/issues/596#issuecomment-1548797780):

> Underneath, bbi init looks through .nonmem_dir and tries to detect subdirectories for different NONMEM versions ([code](https://github.com/metrumresearchgroup/bbi/blob/f8a2c83869381a2d1221aeb302da880093079b91/cmd/init.go#L39-L76)). Under each of those subdirectories, it looks for things like expected subdirectories, a license file, and an NONMEM executable.

I think the example layout would definitely help alot, but was worried we wouldnt want to mention Metworx. Since you posted that example without mentioning Metworx ill definitely add that. I do think it adds clarity to mention the executable, or some file that we can point to and know it's 2 directories up from there. I'll try to rephrase and potentially remove that word altogether, but I thought your comment on that issue added some clarity

Reply:
> I was trying to reword your comment [...]

I see.  Yeah, the original comment is trying to distinguish a couple levels of directories (e.g., "Under of of those subdirectories ..."), and I think in the comment here it'd be unclear to me that it's talking about subdirectories within the NONMEM installation directory.

> an NONMEM executable.

Aside from my "an" typo, yeah, I was a bit sloppy there to say "NONMEM executable" and since I was describing in detail what bbi does, I should have specified `nmfe*`.  Again, in my eyes that a bit too much detail for here, but I'm okay with what you decide there.  The main thing I'm trying to avoid here is statements like "`.nonmem_dir` should therefore point to the *parent* directory containing the executable"; I have a hard time making that correct unless "containing" is interpreted recursively (which I think would be surprisingly).

Reply:
I only have one mention of the executable in my most recent commit. Let me know if it be helpful to add `nmfe` before that mention, or if we should reword that sentence altogether. I kind of like mentioning those contents in the event external scientists are more familiar with specific NONMEM installation files, but definitely want it to be as obvious as possible